lib/workload/stateless/stacks/filemanager/deploy/lib/filemanager.ts

```diff
@@ -53,36 +53,57 @@ export class Filemanager extends Stack {
   constructor(scope: Construct, id: string, props: FilemanagerProps) {
     super(scope, id, props);
 
+    const vpc = Vpc.fromLookup(this, 'MainVpc', props.vpcProps);
+
+    const lambdaSecurityGroup = SecurityGroup.fromLookupByName(
+      this,
+      'OrcaBusLambdaSecurityGroup',
+      props.securityGroupName,
+      vpc
+    );
+
     const host = StringParameter.valueForStringParameter(
       this,
       props.databaseClusterEndpointHostParameter
     );
 
     if (props?.migrateDatabase) {
       const migrateFunction = new MigrateFunction(this, 'MigrateFunction', {
-        vpc: props.vpc,
+        vpc: vpc,
         host: host,
         port: props.port,
-        securityGroup: props.securityGroup,
-        buildEnvironment: props?.buildEnvironment,
-        rustLog: props?.rustLog,
+        securityGroup: lambdaSecurityGroup,
+        buildEnvironment: {},
+        // rustLog: props?.rustLog,
```

Comment:
You can remove these two properties, currently they aren't being used but I'll add back in if needed.